tests/integration/e2e/trade_demo_smpc_adp_test.py

```diff
@@ -48,7 +57,7 @@ def test_end_to_end_smpc_adp_trade_demo() -> None:
 
     # Canada
     ca_root = sy.login(email="info@openmined.org", password="changethis", port=9082)
-    ca_data = load_data(csv_file="ca - feb 2021.csv")
+    ca_data = pd.read_csv(download_trade_dataset(csv_file="ca - feb 2021.csv"))[0:10]
```

Comment:
@rasswanth-s we can simply this by just passing the dataset URL to the `read_csv` method.
```
dataset_url = "https://raw.githubusercontent.com/openmined/datasets/main/trade_flow/ca%20-%20feb%202021.csv"
ca_data = pd.read_csv(dataset_url)
```

Reply:
Modified in [efa5da7](https://github.com/OpenMined/PySyft/pull/7165/commits/efa5da7b9aac04a2ce7f3dbeb5ca2c460c1814f7)